packages/fluentui/styles/src/mergeThemes.ts

```diff
@@ -166,35 +165,40 @@ export const mergeComponentStyles: (
 /**
  * Merges a single component's variables with another component's variables.
  */
-export const mergeComponentVariables__PROD = (...sources: ComponentVariablesInput[]): ComponentVariablesPrepared => {
-  const initial = () => ({});
-
-  // filtering is required as some arguments can be undefined
-  const filteredSources = sources.filter(Boolean);
+export const mergeComponentVariables__PROD = (
+  variablesA: ComponentVariablesInput | undefined,
+  variablesB: ComponentVariablesInput | undefined,
+): ComponentVariablesPrepared => {
+  if (variablesA && variablesB) {
+    return function mergedComponentVariables(...args) {
```

Comment:
Do we need rest here? By the typings this will always be a single argument.

Reply:
No, we don't 👍 Changed it in dadbd91